### PR TITLE
feat: add tabs component from oruga

### LINKED ISSUE #4081: Went from collections to gallery on Basilisk
No error in the network console 

image

image

### LINKED ISSUE #4232: redesign: Tabs Component + Properties/Description/Details
requirements: 
- https://github.com/kodadot/nft-gallery/issues/4111#properties-section
- https://github.com/kodadot/nft-gallery/issues/4111#description-section
- https://github.com/kodadot/nft-gallery/issues/4111#details-section

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [x] Refactoring

## Context

- [x] Closes #4232
- [x] Closes #4081. How to reproduce:
  1. visit gallery item (direct link) https://kodadot.xyz/rmrk/gallery/11823638-900D19DC7D3C444E4C-KSMBOT-KUSAMA_BOT_063-0000000000000066
  2. navigate to Gallery under Explore > Gallery menu

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [x] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by yangwao** [comment]
What edge cases have you found?

File: components/gallery/GalleryItemDescription.vue [hunk 1]
```diff
@@ -0,0 +1,114 @@
+<template>
+  <o-tabs v-model="activeTab" expanded>
+    <o-tab-item value="0" label="Properties" class="py-4">
+      <table class="simple-table">
```
Comment (roiLeo) [inline components/gallery/GalleryItemDescription.vue:4]: Why don't you use `<o-table>`?

File: components/gallery/useGalleryItem.ts [hunk 2]
```diff
@@ -25,35 +45,27 @@ export const useGalleryItem = () => {
...
+      return
     }
 
-    if (nftMeta?.id) {
-      await fetchNFTMetadata(nftMeta)
+    nft.value = nftEntity
+
+    if (!nftEntity.metadata) {
+      $consola.log(`NFT with id ${params.id} has no metadata`)
+      return
```
Comment (roiLeo) [inline components/gallery/useGalleryItem.ts:60]: Do we really need to have empty return statement?

File: components/gallery/GalleryItem.vue [hunk 3]
```diff
@@ -1,14 +1,14 @@
 <template>
-  <section class="py-5">
+  <section class="py-5 gallery-item">
     <div class="columns">
-      <div class="column is-two-fiths">
+      <div class="column is-two-fifths">
```
Comment (roiLeo) [inline components/gallery/GalleryItem.vue:4]: hehe nice catch

**Review by roiLeo — COMMENTED** [review]
- we need to hide "Offers" tab on rmrkv1
- Darkmode don't have bg color
Screenshot 2022-11-14 at 11-21-52 KodaDot - Kusama NFT Market Explorer

File: package.json [hunk 4]
```diff
@@ -73,6 +73,7 @@
     "@nuxtjs/apollo": "^4.0.1-rc.5",
     "@nuxtjs/i18n": "^7.3.0",
     "@nuxtjs/sentry": "^6.0.1",
+    "@oruga-ui/oruga": "^0.5.8",
```
Comment (roiLeo) [inline package.json:76]: wouldn't it be `@oruga-ui/oruga-next`?

Comment (roiLeo) [reply package.json:76]: Yep made some test and oruga-next wasn't working with our current setup.
But there is still something that I'm missing out, if we integrate oruga to histoire then we won't need the dependency in the root of the project?

**Review by roiLeo — APPROVED** [review]
Do you know if this new dependencie will make the application slower?
IMO, we can move forward since we need oruga-ui to create new components

// TODO:
- missing translation
- histoire component

**Review by kkukelka — APPROVED** [review]

**Comment by kkukelka** [comment]
there were no bounty tags attached to the tickets. was that intended? 👀

**Pull request merged**